provisioner/roles/manage_ec2_instances/tasks/instances_security.yml

```diff
@@ -0,0 +1,164 @@
+- name: find ami for splunk
```

Comment:
Should we not put this find-ami-code into its own task? Just like we did it with the networking find-ami stuff?
@IPvSean, comments?

Reply:
I am flexible in this... I try to divide up the Ansible Playbook into 150 or less lines so they are human readable.  This Ansible Playbook is still not that long... but it would be nice to match?  I feel like the way I organize this is getting confusing and I would be OK with rethinking how we organize this.  @maxamillion do you have an opinion?

Reply:
it's possible the "find AMI" tasks might need to live in their own task file at this point and be included at the appropriate point in the `main.yml` for clarity ... this file is approaching unwieldy status, it's definitely not there yet but as we grow the workshop more I could see us getting there without too much effort